metaseq_cli/train.py

```diff
@@ -418,6 +418,10 @@ def validate_and_save(
             and num_updates >= cfg.dataset.validate_after_updates
             and was_successful_step
         )
+        or (
+            cfg.dataset.log_training_trajectory
```

Comment:
why do we need to save? Just validate?

Reply:
I want to plot the trajectory for generation tasks as well (ROUGE); eval will be fast